packages/mantine/package.json

```
@@ -0,0 +1,97 @@
+{
+  "name": "@rjsf/mantine",
+  "version": "6.0.0-alpha.0",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
```

You will need to add this to match the other theme's improved ESM support:
```suggestion
  "typings": "lib/index.d.ts",
  "type": "module",
  "exports": {
    ".": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./lib": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./lib/*.js": {
      "require": "./dist/*.js",
      "import": "./lib/*.js",
      "types": "./lib/*.d.ts"
    },
    "./dist": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./dist/*.js": {
      "require": "./dist/*.js",
      "import": "./lib/*.js",
      "types": "./lib/*.d.ts"
    }
  },
```